k9mail/src/main/java/com/fsck/k9/activity/K9ListActivity.java

```diff
@@ -1,20 +1,22 @@
 package com.fsck.k9.activity;
 
-import android.app.ListActivity;
 import android.os.Bundle;
 import android.view.KeyEvent;
 import android.view.MotionEvent;
 import android.widget.AdapterView;
+import android.widget.ListAdapter;
 import android.widget.ListView;
 
 import com.fsck.k9.K9;
 import com.fsck.k9.activity.K9ActivityCommon.K9ActivityMagic;
 import com.fsck.k9.activity.misc.SwipeGestureDetector.OnSwipeGestureListener;
 
 
-public abstract class K9ListActivity extends ListActivity implements K9ActivityMagic {
+public abstract class K9ListActivity extends K9Activity implements K9ActivityMagic {
 
     private K9ActivityCommon mBase;
+    protected ListAdapter mAdapter;
```

Comment:
Please don't create new fields with the `m` prefix. I'd rather mix than introduce more of these.

Reply:
I'll submit a review to get rid of them and rebase.